### DISCUSSION TITLE
copy as an image

### DISCUSSION THREAD

**Discussion by edbeee** [freeplane/freeplane#2052] [Help]
How can I do "copy as an image" of selected branches? is this available? is there any script for that? does anyone have any idea regarding this?
![image](https://github.com/user-attachments/assets/cf442017-93e4-4893-b40d-123bbb978863)

**Comment by euu2021** [comment]
Similar to https://github.com/freeplane/freeplane/issues/733

I think it's possible to create a script for that, by using methods similar to the ones of ExportToImage class: https://github.com/freeplane/freeplane/blob/42e79bda67848e14d1742ba9f9b87b86990076a9/freeplane/src/main/java/org/freeplane/features/export/mindmapmode/ExportToImage.java#L55

This is not easy, though.

**Comment by dpolivaev** [comment]
I think you can jump in into the branch root node and export the image

**Reply by dpolivaev** [reply]
but it does not work for multiple branches.
For  multiple branches you can use "filter" "selected nodes" "with ancestors"
